pkg/network/protocols/http2/protocol.go

```diff
@@ -129,7 +129,7 @@ func (p *protocol) Name() string {
 }
 
 const (
-	mapSizeValue = 1024
+	mapSizeValue = 10240
```

Comment:
will be changed as part of this [PR](https://github.com/DataDog/datadog-agent/pull/20955/files)

Reply:
good catch!